Runtime/Internal/Serialization/SharedCore.cs

```diff
@@ -31,6 +30,7 @@ internal static void Flush(bool copyToCache)
             SerializeGaze(copyToCache);
             SerializeSensors(copyToCache);
             SerializeFixations(copyToCache);
+            SerializeBoundaryShapes();
```

Comment:
This should have an optional flag to write the data to the local cache (if we can't confirm there will be a response from the server)

Reply:
Done